helm-charts/karpenter-resources/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
 name: karpenter-resources
 description: Helm chart for configuring custom resources for Karpenter on the cluster
-version: 0.0.1
-appVersion: 0.0.1
+version: 0.0.2
+appVersion: 0.0.2
```

Comment:
Changing this version might not make a difference since we are not publishing these Helm charts. You can keep it as 0.0.1.

I will create a new tag for the Data Add-ons, so that existing blueprints will still work with the old tags, and the new ones will need updating to below.

```hcl
  source  = "aws-ia/eks-data-addons/aws"
  version = "~> 1.40" 
```

Reply:
reverted back to 0.0.1